.github/workflows/auto-changelog.yml

```
@@ -0,0 +1,73 @@
+              credits: ["@dependabot[bot]"]
+            }' > "$FILE"
+
+          # jq の整形は Prettier と一致しないため、マージ後 main の
+          # format:check が落ちないようにリポジトリと同じ Prettier で整形する
+          npx --yes "prettier@$(node -p 'require("./package.json").devDependencies.prettier')" --write "$FILE"
+
+      - name: Commit
+        env:
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          PR_NUMBER: ${{ github.event.pull_request.number }}
+        run: |
```

actionlint (1.7.12)

[error] "github.event.pull_request.head.ref" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/reference/security/secure-use#good-practices-for-mitigating-script-injection-attacks for more details

(expression)